### ISSUE TITLE
Unable to move transitions dialog on with Jaws and NVDA

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/models-of-the-hydrogen-atom#157]
**Device**
Dell Laptop
**OS**
Win 11
**Browser**
Chrome + Jaws, Firefox + NVDA
**Problem Description**
For https://github.com/phetsims/qa/issues/1214. I hadn't thought to check this until @Nancy-Salpepi mentioned https://github.com/phetsims/models-of-the-hydrogen-atom/issues/153. With Jaws and NVDA, if you try to use arrow keys or "awsd" to move the transitions dialog box, the Jaws or NVDA commands overwrite them. So I have not found a way while those screen readers are active to use keyboard nav to move the dialog.

**Comment by jessegreenberg** [comment]
Discussed during design meeting today, we will fix this one.

The dialog role keeps the screen reader in document mode, I think this will be fixed by adding an application to the dialog container so that keyboard events are received.

**Comment by Nancy-Salpepi** [comment]
I wanted to note that I was able to move the dialog when VO was on with my mac, but not by using VO keys. I moved it using only the arrow keys.

**Comment by pixelzoom** [comment]
https://github.com/phetsims/models-of-the-hydrogen-atom/issues/157#issuecomment-2704976315 is Greek to me, so @jessegreenberg please make whatever changes are necessary. Then assign to me for review.

**Comment by jessegreenberg** [comment]
I changed my mind about this. We can make the "dialog" draggable but that would prevent the user from reading the dialog content with the virtual cursor (unless they manually go back to browse mode). That makes it less accessible. If a user wants to move it they can still manually enter forms mode to do so.

I discussed with @terracoda and @KatieWoe to confirm that this is correct.
